### DISCUSSION TITLE
Route not working

### DISCUSSION THREAD

**Discussion by nehagaur89** [pimcore/pimcore#15782] [Q&A / Support] [answered]
Hello All, I need one help.  I am not able to get my route working. I have added the code for route in the bundle. I have tried static routes also , still not working.

![Screenshot from 2023-08-21 16-53-03](https://github.com/pimcore/pimcore/assets/123084013/3d6478e0-4e53-4f91-aaac-087c224098cb)
![Screenshot from 2023-08-21 16-54-12](https://github.com/pimcore/pimcore/assets/123084013/83537a54-7bac-4456-8621-09bf0a8f466e)

**Comment by jdreesen** [comment, marked as answer]
You have to expose it to be able to use it in JavaScript: https://github.com/FriendsOfSymfony/FOSJsRoutingBundle/blob/master/Resources/doc/usage.rst#generating-uris

**Comment by nehagaur89** [comment]
Many thanks , it helped me a lot in knowing and debugging for what could be wrong , at the end **, options={"expose" = true}** adding this at the end of annotation helped. Thank you so much once again.
